app/views/users/index.html.erb

```diff
@@ -4,8 +4,38 @@
       <li>
         Name: <%= user.name %>
         <span class="profile-link">
-          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          <%= link_to 'See Profile', user_path(user), class: 'profile-link', id: "user-#{user.id}" %>
         </span>
+        <% if @logged_user.friend_request_sent?(user) %>
```

Comment:
- [ ] View-related logic can be encapsulated in View helpers (app/helpers/*_helper.rb)

- [ ] There shoud be absolutely NO logic in views.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png)

Let's move all the logic to its corresponding helpers.

Reply:
Solved too, thanks @mosaaleb